benchmark/webstreams/creation.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const common = require('../common.js');
+const {
+  ReadableStream,
+  TransformStream,
+  WritableStream,
+} = require('node:stream/web');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  n: [50e3],
+  kind: ['ReadableStream', 'TransformStream', 'WritableStream']
+});
+
+let rs, ws, ts;
+
+function main({ n, kind }) {
+  switch (kind) {
+    case 'ReadableStream':
+      bench.start();
+      for (let i = 0; i < n; ++i)
+        rs = new ReadableStream();
+      bench.end(n);
+
+      // Avoid V8 deadcode assignment
```

Comment:
Does this read "this assignment avoids V8 deadcode (elimination)"?

Reply:
yes, let me adjust.